src/lib/curl/base.c

```diff
@@ -72,6 +72,15 @@ CONF_PARSER fr_curl_tls_config[] = {
 	CONF_PARSER_TERMINATOR
 };
 
+CONF_PARSER fr_curl_conn_config[] = {
+	{ FR_CONF_OFFSET("min", FR_TYPE_UINT32, fr_curl_conn_config_t, min_elements), .dflt = "10" },
+	{ FR_CONF_OFFSET("max", FR_TYPE_UINT32, fr_curl_conn_config_t, max_elements), .dflt = "100" },
```

Comment:
This is a little confusing.  We have `max` and then we have `limit`... So max isn't really the maximum.  It's more like `pool_min`, `pool_max`, or `cache_min`, `cache_max`.  I don't know @alandekok  any comments?  The basic issue here is when `limit = yes` we limit the number of connections to `max`, if `limit = no` the only limit comes from the limit sections in unlang.  I think we might not want to expose limit at all, and ONLY allow limiting from unlang, and keep the limit arg for internal uses.